src/read/cfi.rs

```diff
@@ -4272,12 +4280,12 @@ mod tests {
         let expected_addr = 0xdead_beef;
         let section = Section::with_endian(Endian::Little)
             .D8(constants::DW_CFA_set_loc.0)
-            .uleb(expected_addr)
+            .L64(expected_addr)
             .append_bytes(&expected_rest);
         let contents = section.get_contents().unwrap();
         let input = &mut EndianSlice::new(&contents, LittleEndian);
         assert_eq!(
-            CallFrameInstruction::parse(input),
+            CallFrameInstruction::parse(input, mem::size_of::<usize>() as u8),
```

Comment:
This address size must match the size of the `expected_addr` that is written, so it must be 8. It would also be worth adding a variant of this test that writes a 32 bit address.